spec/spec_helper.rb

```diff
@@ -62,6 +62,7 @@ def use_cassette(name, options={})
   # Set VCR_RECORD=once to re_record
   record_option = ENV.fetch("VCR_RECORD") { "none" }.to_sym
   options = {record: record_option }.merge!(options)
+  name = "#{creds_type}/#{name}" unless name =~ /oauth/
```

Comment:
how the creds recordings are split. Notice some are *always* oauth1 or oauth2, so it doesn't rename those